### ISSUE TITLE
Error loading devices: cannot read properties of undefined

### ISSUE THREAD

**Issue by alexbde** [PhilippEngler/eufy-security-hm#9]
**Describe the bug**
After upgrading from version 2.0.1 to 2.1.0 yesterday, I receive the error `Error occured at getDevicesAsJson(). Error: Cannot read properties of undefined (reading 'start_time')` while trying to view the devices page. I also see a lot of those errors in the log (see below). Additionally, the system variable `eufyLastConnectionResult` states `fehlerhaft`.

**To Reproduce**
1. Go to the eufy security addons page > devices/Geräte
2. Scroll down to devices/Geräte
3. See the error

**Expected behavior**
Devices are shown with their status.

**Screenshots**
<img width="646" alt="Bildschirmfoto 2023-05-23 um 10 16 25" src="https://github.com/PhilippEngler/eufy-security-hm/assets/17784818/e29bee7e-bc66-4111-8a91-d396cf9b2e3a">

<img width="1186" alt="Bildschirmfoto 2023-05-23 um 10 24 58" src="https://github.com/PhilippEngler/eufy-security-hm/assets/17784818/3a4b9da3-bd6f-4f6c-9f94-4a04e6a6d996">

**Desktop/Mobile Device (please complete the following information):**
All devices.

**CCU/RaspberryMatic (please complete the following information):**
 - Device: CCU3
 - OS: 3.69.7
 - Installed AddOns: XML-API, NEO SERVER, eufy SECURITY

**Additional context**
Restarting the addon solves the issue for ~ 15 minutes. Afterwards, the error occurs again. Here's the full logs since the last restart yesterday 8:25 pm.

<details>
<summary>stdout/Protokolldatei</summary>

```
2023-05-22 20:25:38 - Going to restart with apiServerRestarter...
2023-05-22 20:25:39 - SIGTERM signal received. Save config and shutdown server...
2023-05-22 20:25:39 - Set service state to shutdown...
2023-05-22 20:25:39 - Stopping scheduled tasks...
2023-05-22 20:25:39 - Remove scheduling for updateDeviceDataData.
2023-05-22 20:25:39 - Remove scheduling for getLibrary.
2023-05-22 20:25:39 - Remove scheduling for saveConfig24h.
2023-05-22 20:25:39 - Stopping EufySecurityApi...
2023-05-22 20:25:39 - Remove scheduling for updateDeviceDataData.
2023-05-22 20:25:39 - Remove scheduling for getLibrary.
2023-05-22 20:25:39 - Remove scheduling for saveConfig24h.
2023-05-22 20:25:39 - Stopping PushService...
2023-05-22 20:25:39 - Push notification connection closed.
2023-05-22 20:25:39 - Stopping MqttService...
2023-05-22 20:25:39 - Closing connections to all stations...
2023-05-22 20:25:42 - Closing connections to all devices...
2023-05-22 20:25:42 - Write config...
2023-05-22 20:25:42 - Stopping...
2023-05-22 20:25:42 - Stopped...
2023-05-22 20:25:42 - ...done. Exiting
2023-05-22 20:25:46 - eufy-security-hm version 2.1.0 (2.6.2)
2023-05-22 20:25:46 -   Platform: linux_arm
2023-05-22 20:25:46 -   Node: v16.19.1
2023-05-22 20:25:46 - Starting http server...
2023-05-22 20:25:46 - ...started. http listening on port '52789'
2023-05-22 20:25:46 - Starting https server...
2023-05-22 20:25:46 - ...started. https listening on port '52790'
2023-05-22 20:25:46 - Started initializing push notification connection.
2023-05-22 20:25:52 - Setting up scheduled tasks...
2023-05-22 20:25:52 -   updateDeviceData scheduled (runs every 10 minutes).
2023-05-22 20:25:52 -   scheduling getState disabled in settings (state changes will be received by event).
2023-05-22 20:25:52 -   getLibrary scheduled (runs every 15 minutes).
2023-05-22 20:25:52 - ...done setting up scheduled tasks.
2023-05-22 20:25:57 - Push notification connection successfully established.
2023-05-23 03:52:56 - Push notification connection closed.
2023-05-23 03:53:02 - Push notification connection successfully established.
2023-05-23 08:56:28 - Push notification connection closed.
2023-05-23 08:56:33 - Push notification connection successfully established.
```
</details>

<details>
<summary>stderr/Fehlerprotokolldatei</summary>

```
2023-05-22 20:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 20:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 21:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 21:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 21:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 21:56:06 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 22:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 22:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 22:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 22:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 23:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 23:26:02 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 23:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-22 23:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 00:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 00:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 00:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 00:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 01:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 01:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 01:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 01:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 02:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 02:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 02:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 02:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 03:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 03:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 03:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 03:52:56 - onSocketError: Error: read ECONNRESET
   at TLSWrap.onStreamRead (node:internal/stream_base_commons:217:20) {
 errno: -104,
 code: 'ECONNRESET',
 syscall: 'read'
}
2023-05-23 03:56:20 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 04:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 04:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 04:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 04:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 05:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 05:26:02 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 05:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 05:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 06:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 06:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 06:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 06:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 07:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 07:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 07:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 07:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 08:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 08:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 08:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 08:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 09:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 09:26:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 09:40:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 09:56:03 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 10:10:53 - Error occured at getLibrary(). Error: Cannot read properties of undefined (reading 'start_time')
2023-05-23 10:12:47 - Error occured at getDevicesAsJson(). Error: Cannot read properties of undefined (reading 'start_time')
```
</details>

**Comment by PhilippEngler** [comment]
Problem already known and will be fixed in next version.

**Comment by alexbde** [comment]
Okay, thank you very much! :) Just leave a comment if I should provide any further information.

**Comment by PhilippEngler** [comment]
Problem is solved with v2.1.1. The images are also loaded after a camera event occurred.
